rivertest/rivertest.go

```diff
@@ -358,47 +359,3 @@ func failure(t testingT, format string, a ...any) {
 func failureString(format string, a ...any) string {
 	return "\n    River assertion failure:\n    " + fmt.Sprintf(format, a...) + "\n"
 }
-
-// WARNING!!!!!
-//
-// !!! When updating this function, the equivalent in `./job.go` must also be
-// updated!!!
-//
-// This is obviously not ideal, but since JobRow is at the top-level package,
-// there's no way to put a helper in a shared package that can produce one,
-// which is why we have this copy/pasta. There are some potential alternatives
-// to this, but none of them are great.
-func jobRowFromInternal(internal *dbsqlc.RiverJob) *river.JobRow {
```

Comment:
Note we get to kill this duplicated convert function from `rivertest` which is nice.